arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,129 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "SALifeTimeSeconds": {
```

Comment:
Okay noted. Although a heads up, it seems that there are some schemas that seem to break this protocol (see processor architecture under vpnclientparameters).